src/Sfa.Das.Sas.Tools.MetaDataCreationTool/Services/AssessmentOrgsExcelPackageService.cs

```diff
@@ -56,6 +56,7 @@ public IEnumerable<StandardOrganisationsData> GetStandardOrganisationsData(Excel
                         EpaOrganisationIdentifier = standardOrganisationWorkSheet.Cells[i, 1].Value != null ? standardOrganisationWorkSheet.Cells[i, 1].Value.ToString() : string.Empty,
                         StandardCode = standardOrganisationWorkSheet.Cells[i, 3].Value != null ? standardOrganisationWorkSheet.Cells[i, 3].Value.ToString() : string.Empty,
                         EffectiveFrom = standardOrganisationWorkSheet.Cells[i, 5].Value != null ? Convert.ToDateTime(standardOrganisationWorkSheet.Cells[i, 5].Value.ToString()) : default(DateTime),
```

Comment:
The default value of EffectiveFrom is DateTime.MinValue

to avoid confusion here, you could use DateTime.MaxValue